ui/admin/app/templates/scopes/scope/workers.hbs

```diff
@@ -4,4 +4,9 @@
 }}
 
 {{page-title (t 'resources.worker.title_plural')}}
+<Breadcrumbs::Item
```

Comment:
None blocking comment: Is this a new way/pattern to define the breadcrumb? I like it 😄 if so, would you mind to present or document somehow (or provide a link if is already documented somewhere) the new way of working with this? (not asking this as part of the PR)

I see we need to define `<Breadcrumbs::Container>` and then the `<Breadcrumbs::Item>` ?

Reply:
This pattern is currently used in cloud-ui and uses this [addon](https://github.com/Bagaar/ember-breadcrumbs). I will add this to tech time topics (to give overview of new pattern). I can also add to docs (a README perhaps), any preference of where it should go?

Reply:
@lisbet-alvarez I think mentioning it in techTime it's enough since you already provided [the documentation](https://github.com/Bagaar/ember-breadcrumbs). Amazing!